src/www/system_advanced_network.php

```diff
@@ -170,7 +180,35 @@
                   </div>
                 </td>
               </tr>
+              <tr>
+                <td width="22%"><strong><?= gettext('DHCP6c DUID') ?></strong></td>
+                <td width="78%" align="right">
+                  <small><?=gettext("full help"); ?> </small>
+                  <i class="fa fa-toggle-off text-danger"  style="cursor: pointer;" id="show_all_help_page" type="button"></i>
+                </td>
+              </tr>
+              <tr>
+                  <td><a id="help_for_fixed_DUID" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Fixed DUID"); ?></td>
+                  <td>
+                    <input name="global-v6duid" type="text" id="global-v6duid" value="<?=htmlspecialchars($pconfig['global-v6duid']);?>" />
```

Comment:
html_safe wraps htmlspecialchars

Reply:
All pconfig values are escaped by default

Reply:
I borrowed that bit from the existing spoofmac code.